src/ConvertToSarif/ConvertOptions.cs

```diff
@@ -10,13 +10,14 @@ namespace Microsoft.CodeAnalysis.Sarif.ConvertToSarif
     internal class ConvertOptions
     {
         [Value(0,
+               MetaName = "<inputLogFile>",
```

Comment:
This shows up in the help output instead of the default string `"value pos. 0`":

```
ConvertToSarif 1.0.0.0
Copyright c  2015

  -t, --tool                 The tool format of the input file. Must be one of:
                             AndroidStudio, ClangAnalyzer, CppCheck, Fortify,
                             FxCop, or PREfast.

  -o, --output               A file path to the converted SARIF log. Defaults
                             to <input file name>.sarif.

  -p, --pretty               (Default: false) Produce pretty-printed JSON
                             output rather than compact form.

  -f, --force                (Default: false) Force overwrite of output file if
                             it exists.

  --help                     Display this help screen.

  --version                  Display version information.

  <inputLogFile> (pos. 0)    A file path to a tool log file that should be
                             converted to the SARIF format.
```